controllers/pwas/crud.js

```diff
@@ -136,7 +136,8 @@ router.post('/:pwa/edit', (req, res, next) => {
 router.get('/:pwa', (req, res, next) => {
   pwaModel.find(req.params.pwa, (err, entity) => {
     if (err) {
-      return next(err);
+      // Not really an error: the pwa wasn't found in the db. Fall through to 404 page.
```

Comment:
Strangely this error comes all the way from https://github.com/GoogleChrome/gulliver/blob/master/lib/model-datastore.js#L175. It appears `get()` does not distinguish between a "Bad Request" and a key not found.

Not sure of the "right" way to fix this--pre-flight the `get()` with a query for the key? But that would be two trips. @juliantoledo do you know?

Anyway, I don't think this is a problem for this PR, will punt for later.